components/ProductCartCard.js

```diff
@@ -9,10 +16,44 @@ import { ProductBody, ProductCardContainer } from '../styled/checkout.js';
 // TODO @tommypoa to use styled-components
 function ProductCartCard({ product }) {
   return (
-    <ProductCardContainer>
-      <ProductBody>{product.name}</ProductBody>
-      <Text>{product.cartCount}</Text>
-    </ProductCardContainer>
+    // <View style={{ alignItems: 'center' }}>
+    <LineItem>
+      <View
+        style={{
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          marginLeft: 24,
+          marginRight: 26,
+          marginTop: 13
+        }}>
+        <LineItemName style={{ justifyContent: 'flex-start' }}>
+          {product.name}
+        </LineItemName>
+        <LineItemPrice style={{ justifyContent: 'flex-end' }}>
+          ${(product.customerCost * product.cartCount).toFixed(2)}
+        </LineItemPrice>
+      </View>
+      <View
+        style={{
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          marginLeft: 24,
+          marginRight: 26
+        }}>
+        <LineItemDetail style={{ justifyContent: 'flex-start' }}>
+          {product.detail}
+        </LineItemDetail>
+        <LineItemPoints style={{ justifyContent: 'flex-end' }}>
+          {product.points * product.cartCount} pts
+        </LineItemPoints>
+      </View>
```

Comment:
General restructuring of each line in this component:
Make a styled component `LineItemRow` that can be reused for the 3 rows of this LineItem. Within each row, directly place base component text styles. Use flex to get the positioning of each row, then use flex in the LineItem container.

Check out this example here that Myles helped with: https://output.jsbin.com/zoyokezemu